ui/src/actions/sampleview.js

```diff
@@ -120,21 +125,21 @@ export function addCentringPoint(x, y) {
   };
 }
 
-export function addShape(shape) {
+export function addShapeAction(shape) {
```

Comment:
Adding suffix `Action` to differentiate this simple action creator from the `addShape` async action creator.